test/algolia_test.exs

```diff
@@ -379,10 +379,16 @@ defmodule AlgoliaTest do
         "type" => "synonym"
       },
       %{
-        "synonyms" => ["tiny"],
+        "synonyms" => ["short"],
         "type" => "oneWaySynonym",
-        "objectID" => "785493758483",
-        "input" => "little"
+        "objectID" => "785493768501",
+        "input" => "small"
+      },
+       %{
+        "synonyms" => ["short"],
+        "type" => "oneWaySynonym",
+        "objectID" => "785493768501",
```

Comment:
Why is this synonym the same as the one above it?

Reply:
Oh, nice catch! I updated by accident the existent synonym.